.ansible-lint

```diff
@@ -1,6 +1,9 @@
 skip_list:
   - yaml
-  - fqcn-builtins  # Use FQCN for builtin actions.
-  - no-handler
-  - command-instead-of-shell
-  - no-jinja-when
+  - 'fqcn[action-core]'
+  - 'fqcn[action]'
+  - 'no-handler'
+  - 'command-instead-of-shell'
+  - 'no-jinja-when'
+  - 'name[casing]'
```

Comment:
Could we move that to the `warn_list`? I think it doesn't need to break a check but if we see it in the overview we can fix it eventually. Personally, I like that rule but I don't want a check to fail because of it. What do you think?

Reply:
Nice idea I will do it :)

Reply:
Done.